src/tools/rest-frontend/public/.htaccess

```diff
@@ -0,0 +1,20 @@
+<IfModule mod_rewrite.c>
+    <IfModule mod_negotiation.c>
+        Options -MultiViews
+    </IfModule>
+
+    RewriteEngine On
+
+    # Fix problem with angular jwt auth
+    RewriteCond %{HTTP:Authorization} ^(.*)
+    RewriteRule .* - [e=HTTP_AUTHORIZATION:%1]
```

Comment:
I'm asking myself where the whole .htaccess comes from, can't remember writing it. :smile: